### ISSUE TITLE
crash when selecting a channel from a removed slack

### ISSUE THREAD

**Issue by coffeejunk** [eul-im/eul#109]
Version: volt v0.55

Repro steps:
1. Add a Slack and let the messages & channels sync
1. Add a second Slack and let the messages & channels sync
1. Remove the second Slack
    - the channels from the removed Slack remain
1. Select one of the channels from the removed Slack
    - Volt crashes silently

Log:
```
2019-02-18 14:38:21.067 volt[37871:28731442] load chat - OK
exiting from os.exit(): slack api not found
2019-02-18 14:38:21.078 volt[37871:28731442] slack api not found
```

**Comment by voltapp2** [comment]
Thanks for the detailed report.

I know what the problem is.

Do you only get the crash in this scenario? No more crashes on launch?

**Comment by coffeejunk** [comment]
Yep, this is the only crash I'm observing right now.

**Comment by voltapp2** [comment]
Great, thanks! I’ll fix this one later today.

**Comment by voltapp2** [comment]
Fixed in 0.81
